Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2Connection.swift

```diff
@@ -164,15 +164,23 @@ final class HTTP2Connection {
         return promise.futureResult
     }
 
-    private func start() -> EventLoopFuture<Int> {
+    func start() -> EventLoopFuture<Int> {
```

Comment:
and we should follow the internal naming pattern here: `start0`. as a small cleanup.

Reply:
changed in https://github.com/swift-server/async-http-client/pull/646/commits/ba76fe445fcd25559a564364229c742db8c63316